testsuite/MDAnalysisTests/topology/test_pdb.py

```diff
@@ -325,3 +326,48 @@ def test_nobonds_error():
 
     with pytest.raises(NoDataError, match=errmsg):
         u.atoms.bonds
+
+
+def test_PDB_charges():
+    """The test checks whether formalcharges attribute are assigned
+    properly given a PDB file with a valid formal charges record.
+    """
+    u = mda.Universe(PDB_charges)
+    formal_charges = np.array([0, 0, 0, 0, 0, 0, 0, -1, 0, 0, 0, 0, 0, 0, 0,
+                               0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0,
+                               0, 0, 0, 0, 0], dtype=int)
+    assert_equal(u.atoms.formalcharges, formal_charges)
+
+
+PDB_charges_nosign = """\
```

Comment:
Could possibly be a fixture but i guess its not recycled, up to you :)

Reply:
I'm going to skip on that, the advantages here are limited, especially given we use declared PDB strings like this everywhere else here.